mos_tests/functions/os_cli.py

```diff
@@ -119,3 +139,106 @@ class Ironic(CLICLient):
 
 class Murano(CLICLient):
     command = 'murano'
+
+
+class Swift(CLICLient):
+    command = 'swift'
+
+    def post(self, name):
+        return self('post {0}'.format(name))
+
+    def list(self, name=None):
+        params = ''
+        if name:
+            params += name
+        output = self('list -l', params=params)
+        if len(output) > 0:
```

Comment:
The problem here that in case of empty output `output.split('\n')` will return list and not just string

```
output = self('list -l', params=name)

In [17]: output
Out[17]: u''

In [18]: output.split('\n')
Out[18]: [u'']
```